protocol/messenger.go

```diff
@@ -4156,12 +4151,6 @@ func (m *Messenger) MarkAllReadInCommunity(communityID string) ([]string, error)
 	if err != nil {
 		return chatIDs, err
 	}
-
-	err = m.syncActivityCenterNotifications(notifications)
```

Comment:
@qfrank the same here.

Reply:
same here. unresolving just to point out.

Reply:
it's here:
https://github.com/status-im/status-go/blob/95048404de53733dd1772b64d628c71a65434532/protocol/messenger_activity_center.go#L436

@igor-sirotin